syft_tensorflow/attributes/attributes.py

```diff
@@ -32,11 +32,12 @@ class TensorFlowAttributes(FrameworkAttributes):
     ALIAS = "tensorflow"
     Tensor = TensorFlowTensor
 
-    def __init__(self, tensorflow: ModuleType, hook: "TensorFlowHook"):
+    def __init__(self, tensorflow: ModuleType, hook: ModuleType):
+        super().__init__(tensorflow, hook)
         # Stash the hook here for global access elsewhere
         self.hook = hook
 
-        # SECTION: List all functions in tf module that we want to overload
+        self.allowed_commands = set()
```

Comment:
Do we need this attribute def here? later in the script we do `self.allowed_commands = self._tensorflow_modules_functions`